tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -2499,7 +2499,7 @@ def test_BuiltinDataset():
         dataset = None
 
         # Case 3: dataset is not available.
-        dataset_name = "fake_name"
+        dataset_name = "fake_name-seeds"
```

Comment:
do we need to append `seeds` here? the dataset is always not available.

Reply:
without `seeds` it will cause `Regex pattern did not match`.